configs/dlstreamer/framework-pipelines/yolov5_pipeline/yolov5s_effnetb0.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+#
+# Copyright (C) 2023 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if [ "$INPUTSRC_TYPE" == "REALSENSE" ]; then
+        decode_pp="! videoconvert ! video/x-raw,format=BGR"
+	# TODO: update with vaapipostproc when MJPEG codec is supported.
+	echo "Not supported until D436 with MJPEG." > /tmp/results/pipeline$cid_count.log
+	exit 2
+fi
+
+DECODE="vaapidecodebin"
+DEVICE="CPU" #GPU|CPU|MULTI:GPU,CPU
+PRE_PROCESS="" #""|pre-process-backend=vaapi-surface-sharing|pre-process-backend=vaapi-surface-sharing pre-process-config=VAAPI_FAST_SCALE_LOAD_FACTOR=1
+
+while :; do
+    case $1 in
+    -h | -\? | --help)
+        show_help
+        exit
+        ;;
+	--decode)
+        if [ "$2" ]; then
+            DECODE=$2
+            shift
+        else
+            echo "using default"
+            # error 'ERROR: "--decode" requires an argument vaapidecodebin|decodebin.'
+        fi
+        ;;
+    --device)
+        if [ "$2" ]; then
+            DEVICE=$2
+            if [ -z "$LOW_POWER" ]; then
+                DEVICE="MULTI:GPU,CPU"
+            fi
+            shift
+        else
+            echo "using default"
+            # error 'ERROR: "--device" requires an argument GPU|CPU|MULTI:GPU,CPU.'
+        fi
+        ;;
+    --pre_process)
+        if [ "$2" ]; then
+            PRE_PROCESS=$2
+            shift
+        else
+            echo "using default"
+            # error 'ERROR: "--pre_process" requires an argument ""|pre-process-backend=vaapi-surface-sharing|pre-process-backend=vaapi-surface-sharing pre-process-config=VAAPI_FAST_SCALE_LOAD_FACTOR=1 .'
```

Comment:
in my opinion, we shouldn't ask user to do this

Reply:
this is primitive of what we will do for user, right now this is only to consolidate the codes in one place, @brian-intel has plan to do more to this for user

Reply:
could we remove this flag and just use if-else logic to take care of that for now?